miasm2/jitter/codegen.py

```diff
@@ -569,23 +580,25 @@ def gen_c(self, block, log_mn=False, log_regs=False):
         if isinstance(block, AsmBlockBad):
             return self.gen_bad_block()
         irblocks_list = self.block2assignblks(block)
-
         out, instr_offsets = self.gen_init(block)
         assert len(block.lines) == len(irblocks_list)
         for instr, irblocks in zip(block.lines, irblocks_list):
             instr_attrib, irblocks_attributes = self.get_attributes(instr, irblocks, log_mn, log_regs)
-
             for index, irblock in enumerate(irblocks):
                 new_irblock = self.ir_arch.irbloc_fix_regs_for_mode(irblock, self.ir_arch.attrib)
-                if new_irblock.label.offset is None:
+                label = new_irblock.loc_key
+                offset = self.ir_arch.symbol_pool.loc_key_to_offset(label)
+                name = self.ir_arch.symbol_pool.loc_key_to_name(label)
+                if offset is None:
                     out.append("%-40s // %.16X %s" %
-                               (str(new_irblock.label.name) + ":", instr.offset, instr))
+                               (str(name) + ":", instr.offset, instr))
```

Comment:
`name` is only used inside the `if` statement, so `name = ` can be moved here.
In addition, `str()` is useless here

Reply:
done